### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by jaz-on**
Minor adjustments: typo correction + small mistakes.

### CODE REVIEW

**Comment by jaz-on** [comment]
Sorry for https://github.com/obsidianmd/obsidian-translations/pull/531 I'm not used to creating PR.

File: fr.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
 			"option-show-line-number": "Afficher le numéro de ligne",
 			"option-show-line-number-description": "Afficher le numéro de ligne dans la marge.",
 			"option-use-tabs": "Utiliser des tabulations",
-			"option-use-tabs-description": "Utilise des tabulations pour indenter lorsque avec la touche \"Tab\". Désactiver cette option pour utiliser des espaces.",
+			"option-use-tabs-description": "Utilise des tabulations pour indenter lorsqu’avec la touche \"Tab\". Désactiver cette option pour utiliser des espaces.",
```
Comment (Write) [inline fr.json:38]: Would prefer using ' (as used everywhere else) instead of ’

File: fr.json [hunk 2]
```diff
@@ -803,7 +803,7 @@
 			"prompt-type-command": "Saisir une commande...",
 			"label-pinned-commands": "Commandes épinglées",
 			"option-add-new-pin": "Nouvelle commande épinglée",
-			"option-add-new-pin-description": "Les commandes épinglées apparaîtront en haut de toutes les commandes lorsque aucune requête de recherche n'est présente."
+			"option-add-new-pin-description": "Les commandes épinglées apparaîtront en haut de toutes les commandes lorsqu’aucune requête de recherche n'est présente."
```
Comment (Write) [inline fr.json:806]: Would prefer using ' (as used everywhere else) instead of ’

**Review by Write — COMMENTED** [review]
Merci pour cette contribution, tout est ok à part l'utilisation du char. ’ au lieu de ’ (pour être tatillon. :p)
--
Thanks for your contribution, everything is ok appart from using ’ char. instead of ' (as used everywhere else)

**Review by Write — APPROVED** [review]

**Comment by jaz-on** [comment]
@Write I have corrected the 2 lines as suggested on my new commit. I used the wrong character my bad.

**Pull request merged**